test/bats/cli-test.bats

```diff
@@ -16,11 +16,26 @@
 load helpers
 
 @test "notation verifier test" {
+    teardown() {
+        run sudo date -s "-2 days"
```

Comment:
what if we fail at line 23 or 26, in this case we'll update date as well?

Reply:
By doing this I assumed that other e2e tests are not affected but I would be looking for better practice.

Reply:
would it be easy to just create a new test suite for tsa testing? Then we can always update and reset the date.

Reply:
I will give this option a try.